### DISCUSSION TITLE
update action doesn't work properly

### DISCUSSION THREAD

**Discussion by takuma-competition** [astariul/github-hosted-pypi#76] [Q&A]
I created my own authenticated pypi using this repository as a reference.
After publishing my own library with register, I used pip and it worked.
However, when I upgraded the version of that library and performed an update, I could no longer search with pip.

Can you give me some hints to solve this?

error
Looking in indexes: https://pypi.org/simple, https://01ive-co-jp.github.io/olive-hosted-pypi/
Collecting pulse-wave-analysis-library==0.0.3
   ERROR: HTTP error 404 while getting https://01ive-co-jp.github.io/olive-hosted-pypi/pulse-wave-analysis-library/git%2Bb%28%27https%3A/github.com/01ive -co-jp/pulse-wave-analysis-library%27@0.0.3#egg=pulse-wave-analysis-library-0.0.3 (from https://01ive-co-jp.github.io/olive- hosted-pypi/pulse-wave-analysis-library/)
ERROR: Could not install requirement pulse-wave-analysis-library==0.0.3 from https://01ive-co-jp.github.io/olive-hosted-pypi/pulse-wave-analysis-library/git%2Bb %28%27https%3A/github.com/01ive-co-jp/pulse-wave-analysis-library%27@0.0.3#egg=pulse-wave-analysis-library-0.0.3 because of HTTP error 404 Client Error: Not Found for url: https://01ive-co-jp.github.io/olive-hosted-pypi/pulse-wave-analysis-library/git%2Bb%28%27https%3A/github.com/01ive -co-jp/pulse-wave-analysis-library%27@0.0.3 for URL https://01ive-co-jp.github.io/olive-hosted-pypi/pulse-wave-analysis-library/git% 2Bb%28%27https%3A/github.com/01ive-co-jp/pulse-wave-analysis-library%27@0.0.3#egg=pulse-wave-analysis-library-0.0.3 (from https:/ /01ive-co-jp.github.io/olive-hosted-pypi/pulse-wave-analysis-library/)

**Comment by astariul** [comment]
Hey there 👋 

Thank you for creating a discussion.

So, from your HTML page, I can see the link is broken : 
```
<a href="git+https://github.com/01ive-co-jp/pulse-wave-analysis-library@0.0.2#egg=pulse-wave-analysis-library-0.0.2">
  0.0.2
 </a>
</div>
<div class="" id="0.0.3" onclick="load_readme('0.0.3', scroll_to_div=true);">
 <a href="git+b('https://github.com/01ive-co-jp/pulse-wave-analysis-library'@0.0.3#egg=pulse-wave-analysis-library-0.0.3">
  0.0.3
 </a>
``` 

In the second `href`, there is some weird `b('...'`. I don't know where it come from... Maybe it's indeed a bug with the github actions.

You can fix the site by removing this weird `b('...'` from the `href` for the version `0.0.3`.

---

Can you let me know what's the input value you used in the github action `update` ? I want to try to reproduce it !
